### PR TITLE
v0.7.0 Build to next app

### PR DESCRIPTION
**Pull request by alexfarrill**
- Introduces .mailing, remote control next app architecture
- Adds `GET /api/render` API for getting rendered HTML given props and template name
- Adds `npx mailing server`, `npx mailing server build`, `npx mailing server start`

addresses #58

### CODE REVIEW

**Comment by alexfarrill** [comment]
What do we want this to be doing?
image

**Comment by psugihara** [comment]
Nice. That's right except the manifest needs to be next to package.json I think. 

That page is expected not to work bc it's still hitting an api route that doesn't exist. We will switch it to hit /api/previews.json and that will grab info from the manifest.

**Comment by alexfarrill** [comment]
Cool, that makes sense.  Pushed up a refactor to get the directory structure right

**Comment by psugihara** [comment]
working on the api modules

**Comment by alexfarrill** [comment]
- [ ] move render route to an API
- [ ] make `npx mailing deploy` work
- [ ] clean up scripts in package.json

**Comment by psugihara** [comment]
fast :)

File: packages/cli/src/pages/previews/[previewClass]/[previewFunction].tsx [hunk 1]
```diff
@@ -13,21 +19,18 @@ export const getStaticPaths = async () => {
     params: Params;
   }[] = [];
 
-  if (process.env.NEXT_PUBLIC_STATIC) {
-    const res = await fetch("http://localhost:3883/previews.json");
```
Comment (psugihara) [inline packages/cli/src/pages/previews/[previewClass]/[previewFunction].tsx:17]: removes some port hardcoding :)

**Comment by psugihara** [comment]
- [ ] ensure that this setup will work when slurping up templates that are in the same directory as .mailing https://github.com/sofn-xyz/mailing/issues/129#issuecomment-1224337794

**Comment by psugihara** [comment]
latest in 0.7.0-next.1

File: packages/cli/src/util/config.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    logPlain(`
+    ███╗   ███╗ █████╗ ██╗██╗     ██╗███╗   ██╗ ██████╗ 
+    ████╗ ████║██╔══██╗██║██║     ██║████╗  ██║██╔════╝ 
+    ██╔████╔██║███████║██║██║     ██║██╔██╗ ██║██║  ███╗
+    ██║╚██╔╝██║██╔══██║██║██║     ██║██║╚██╗██║██║   ██║
+    ██║ ╚═╝ ██║██║  ██║██║███████╗██║██║ ╚████║╚██████╔╝
+    ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝╚══════╝╚═╝╚═╝  ╚═══╝ ╚═════╝ 
+    
+        `);
```
Comment (mrlubos) [inline packages/cli/src/util/config.ts:75]: The important change

Comment (psugihara) [reply packages/cli/src/util/config.ts:75]: needs design review from @monicakogler at some point ;)

Comment (alexfarrill) [reply packages/cli/src/util/config.ts:75]: Behold!

**Review by psugihara — APPROVED** [review]

**Pull request merged**